### PR TITLE
Fix broken language service (#394)

### PR DESCRIPTION
**Pull request by DaeunYim**

### CODE REVIEW

**Review by pradeepkintali — APPROVED** [review]

File: ossdbtoolsservice/workspace/contracts/common.py [hunk 1]
```diff
@@ -93,6 +93,14 @@ def get_child_serializable_types(cls):
     def __init__(self):
         self.text_document: TextDocumentIdentifier = None
         self.position: Position = None
+
```
Comment (nasc17) [inline ossdbtoolsservice/workspace/contracts/common.py:96]: Whitespace here?

File: ossdbtoolsservice/workspace/contracts/common.py [hunk 2]
```diff
@@ -93,6 +93,14 @@ def get_child_serializable_types(cls):
     def __init__(self):
         self.text_document: TextDocumentIdentifier = None
         self.position: Position = None
+    
+    @classmethod
+    def ignore_extra_attributes(cls):
+        return True
+
+    @classmethod
+    def ignore_extra_attributes(cls):
```
Comment (nasc17) [inline ossdbtoolsservice/workspace/contracts/common.py:102]: Why duplicate methods?

File: ossdbtoolsservice/workspace/contracts/common.py (hunk 1 above)
Comment (DaeunYim) [reply ossdbtoolsservice/workspace/contracts/common.py]: happened by mistake while merging. Thanks!

File: ossdbtoolsservice/workspace/contracts/common.py (hunk 2 above)
Comment (DaeunYim) [reply ossdbtoolsservice/workspace/contracts/common.py:98]: happened by mistake while merging. Thanks!

**Pull request merged**